tests/CMakeLists.txt

```diff
@@ -37,4 +37,5 @@ endif()
 include_directories(${GTEST_INCLUDE_DIRS})
 target_link_libraries(dummy_test ${GTEST_BOTH_LIBRARIES})
 
-add_subdirectory(carddatabase)
\ No newline at end of file
+add_subdirectory(carddatabase)
+add_subdirectory(loading_from_clipboard)
```

Comment:
Yep, I'd like to see each of the tests living inside of their modules eventually too - `cockatrice/tests/carddatabase_test.cpp` vs `tests/carddatabase_test.cpp` and `common/tests/loading_from_clipboard.cpp` vs `tests/loading_from_clipboard.cpp`.